rest_framework/locale/pt/LC_MESSAGES/django.po

```diff
@@ -19,421 +19,421 @@ msgstr ""
 #: authtoken/models.py:15
 msgid "Key"
-msgstr ""
+msgstr "Chave"
 
 #: authtoken/models.py:18
 msgid "User"
-msgstr ""
+msgstr "Utilizador"
 
 #: authtoken/models.py:20
 msgid "Created"
-msgstr ""
+msgstr "Criado"
 
 #: authtoken/models.py:29
 msgid "Token"
-msgstr ""
+msgstr "Token"
 
 #: authtoken/models.py:30
 msgid "Tokens"
-msgstr ""
+msgstr "Tokens"
 
 #: authtoken/serializers.py:8
 msgid "Username"
-msgstr ""
+msgstr "Nome de utilizador"
 
 #: authtoken/serializers.py:9
 msgid "Password"
-msgstr ""
+msgstr "Palavra-passe"
 
 #: authtoken/serializers.py:20
 msgid "User account is disabled."
-msgstr ""
+msgstr "A conta do utilizador está desativada."
 
 #: authtoken/serializers.py:23
 msgid "Unable to log in with provided credentials."
-msgstr ""
+msgstr "Não foi possível fazer login com as credenciais fornecidas."
 
 #: authtoken/serializers.py:26
 msgid "Must include \"username\" and \"password\"."
-msgstr ""
+msgstr "Deve incluir \"nome de utilizador\" e \"palavra-passe\"."
 
 #: exceptions.py:49
 msgid "A server error occurred."
-msgstr ""
+msgstr "Ocorreu um erro no servidor."
 
 #: exceptions.py:84
 msgid "Malformed request."
-msgstr ""
+msgstr "Pedido malformado."
 
 #: exceptions.py:89
 msgid "Incorrect authentication credentials."
-msgstr ""
+msgstr "Credenciais de autenticação incorretas."
 
 #: exceptions.py:94
 msgid "Authentication credentials were not provided."
-msgstr ""
+msgstr "As credenciais de autenticação não foram fornecidas."
 
 #: exceptions.py:99
 msgid "You do not have permission to perform this action."
-msgstr ""
+msgstr "Não tem permissão para realizar esta ação."
 
 #: exceptions.py:104 views.py:81
 msgid "Not found."
-msgstr ""
+msgstr "Não encontrado."
 
 #: exceptions.py:109
 msgid "Method \"{method}\" not allowed."
-msgstr ""
+msgstr "Método \"{method}\" não permitido."
 
 #: exceptions.py:120
 msgid "Could not satisfy the request Accept header."
-msgstr ""
+msgstr "Não foi possível satisfazer o cabeçalho Accept do pedido."
 
 #: exceptions.py:132
 msgid "Unsupported media type \"{media_type}\" in request."
-msgstr ""
+msgstr "Tipo de mídia não suportado \"{media_type}\" no pedido."
 
 #: exceptions.py:145
 msgid "Request was throttled."
-msgstr ""
+msgstr "Pedido foi restringido."
 
 #: fields.py:269 relations.py:206 relations.py:239 validators.py:98
 #: validators.py:181
 msgid "This field is required."
-msgstr ""
+msgstr "Este campo é obrigatório."
 
 #: fields.py:270
 msgid "This field may not be null."
-msgstr ""
+msgstr "Este campo não pode ser nulo."
 
 #: fields.py:608 fields.py:639
 msgid "\"{input}\" is not a valid boolean."
-msgstr ""
+msgstr "\"{input}\" não é um booleano válido."
 
 #: fields.py:674
 msgid "This field may not be blank."
-msgstr ""
+msgstr "Este campo não pode ficar em branco."
 
 #: fields.py:675 fields.py:1675
 msgid "Ensure this field has no more than {max_length} characters."
-msgstr ""
+msgstr "Certifique-se de que este campo não tenha mais do que {max_length} caracteres."
```

Comment:
Django uses "garanta" for "ensure" - perhaps best to be consistent with that?

https://github.com/django/django/blob/78ae8cc5d8aff65afbf35947421a8a1aba13bfab/django/conf/locale/pt/LC_MESSAGES/django.po#L670

Also it doesn't use many "-se" forms - I don't know enough Portuguese to make the style decision here, but perhaps we should follow Django too?

This applies to all the "Certifique-se" messages below...